shiny/ViewMet/server.R

```diff
@@ -141,7 +141,7 @@ server <- function(input, output, session) {
       
       site = query.site(con = bety$con, siteid)
       
-      vars_in_file <- ncdf4::nc_open(rv$load.paths[i]) %>% ncdf4.helpers::nc.get.variable.list()
+      vars_in_file <- ncdf4::nc_open(rv$load.paths[i]) %>% names(nc[[]])
```

Comment:
I think this needs to be:

```suggestion
      vars_in_file <- ncdf4::nc_open(rv$load.paths[i]) %>% names(.[["var"]])
```